docs/getting_started.md

```diff
@@ -3,14 +3,16 @@ All you need to get started building Textual apps.
 
 ## Requirements
 
-Textual requires Python 3.9 or later (if you have a choice, pick the most recent Python). Textual runs on Linux, macOS, Windows and probably any OS where Python also runs.
+Textual requires Python 3.9 or later (if you have a choice, pick the most recent Python). Textual runs on Linux, macOS, Windows and probably any OS where Python also runs. You also need a unicode font that includes the line drawing and block characters, which are used to render the controls.
 
 !!! info "Your platform"
 
     ### :fontawesome-brands-linux: Linux (all distros)
 
     All Linux distros come with a terminal emulator that can run Textual apps.
 
+    The linux console is more limited and not all fonts are suitable. See [Linux console]](./linux-console.md) for details. 
```

Comment:
May devs will not know the difference between the linux console and the terminal on their desktop.

Suggest clarification:

"If you are using the Linux console (rather than a desktop environment), see Linux Console ...]"